go.mod

```diff
@@ -28,27 +28,29 @@ require (
 	github.com/operator-framework/api v0.22.0
 	github.com/operator-framework/operator-lifecycle-manager v0.27.1-0.20240301195430-1d12f8f4de16
 	github.com/rh-ecosystem-edge/kernel-module-management v0.0.0-20240214075243-67ea06a82ab8
-	golang.org/x/exp v0.0.0-20231127185646-65229373498e
-	golang.org/x/net v0.23.0
+	golang.org/x/exp v0.0.0-20240213143201-ec583247a57a
+	golang.org/x/net v0.24.0
 	gopkg.in/k8snetworkplumbingwg/multus-cni.v4 v4.0.2
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.28.7
-	k8s.io/apiextensions-apiserver v0.28.5
-	k8s.io/apimachinery v0.29.0
+	k8s.io/api v0.29.1
+	k8s.io/apiextensions-apiserver v0.29.1
+	k8s.io/apimachinery v0.29.1
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/kubectl v0.28.5
 	k8s.io/kubelet v0.27.4
-	k8s.io/utils v0.0.0-20231127182322-b307cd553661
+	k8s.io/utils v0.0.0-20240102154912-e7106e64919e
 	maistra.io/api v0.0.0-20230704084350-dfc96815fb16
 	open-cluster-management.io/governance-policy-propagator v0.12.0
 	open-cluster-management.io/multicloud-operators-subscription v0.11.0
-	sigs.k8s.io/controller-runtime v0.16.3
+	sigs.k8s.io/controller-runtime v0.17.1
 )
 
 require (
+	github.com/kedacore/keda-olm-operator v0.0.0-20240501182040-762f6be5a942
```

Comment:
Please use v2.12.0 in order to avoid dependency conflict.

Reply:
go: errors parsing go.mod:
go.mod:51:2: require github.com/kedacore/keda-olm-operator: version "v2.12.0" invalid: should be v0 or v1, not v2

Reply:
it is not "github.com/kedacore/keda/v2/apis/keda/v1alpha1",
but ""github.com/kedacore/keda-olm-operator/apis/keda/v1alpha1""

Reply:
it's a different repo